src/client/java/minicraft/screen/LoadingDisplay.java

```diff
@@ -88,7 +94,8 @@ public void render(Screen screen) {
 		super.render(screen);
 		int percent = Math.round(percentage);
 		Font.drawParagraph(screen, new FontStyle(Color.RED), 6,
-			Localization.getLocalized(msg) + (progressType.length() > 0 ? " " + Localization.getLocalized(progressType) : "") + ellipsis.updateAndGet(),
+			Localization.getLocalized(msg) + (progressType.length() > 0 ? " " + (localizeProgressType ? Localization.getLocalized(progressType) : progressType) : "")
+				+ ellipsis.updateAndGet(),
 			percent + "%"
 		);
 	}
```

Comment:
Progress type should just be an enum that is automatically localised.

Reply:
It is not needed to create a new enum class just for that.

Reply:
It appears that it was like that to begin with and was just converted to a “?” operation.